### PR TITLE
[Widget Mode] Create extra-width mode that levelbuilders can use to create widgets

### PR DESCRIPTION
**Pull request by jmkulwik**
Widget mode is an option levelbuilders can use to create a non-resizable version of Applab with extra width. There is no coding or design workspace in widget mode.

**1024x768 screen resolution**
image

**2256x1504 screen resolution, zoomed to 150% (default settings) on a large external monitor**
image

Next steps: 
1. Adjust levelbuilder option to auto-disable code mode, sharing, and design mode.
2. Don't allow widget mode and embed mode to be set at the same time.

Task: https://codedotorg.atlassian.net/browse/STAR-2
Spec: https://docs.google.com/document/d/1laYGDKIgW405YrnrWcURJDAhaKXeQPHrJ7BhyUqJMQE/edit#

### CODE REVIEW

File: apps/src/applab/applab.js [hunk 1]
```diff
@@ -147,7 +147,10 @@ function loadLevel() {
   // This led to lots of hackery in the code to properly scale the visualization
```
Comment (ajpal) [inline apps/src/applab/applab.js:147]: Do you think adding widget mode will re-introduce any problems like this?

**Comment by ajpal** [comment]
Looks great! Worth adding unit tests anywhere?

File: apps/src/applab/applab.js (hunk 1 above)
Comment (jmkulwik) [reply apps/src/applab/applab.js:147]: Normally, yes, but we dodge that bullet here by not allowing resizing. The hackery that this comment is referring to is a series of work-arounds that allow dynamic resizing. For widget mode, we use fixed widths of 600px.

More details are captured in a slack conversation starting here: https://codedotorg.slack.com/archives/C0T0UQPLZ/p1564511130033400

**Comment by jmkulwik** [comment]
> Looks great! Worth adding unit tests anywhere?

Yes. Although I'd like to get this out so levelbuilders can start developing for an upcoming pilot. I'll add the tests as a follow-up.

**Review by ajpal — APPROVED** [review]

**Comment by ajpal** [comment]
> > Looks great! Worth adding unit tests anywhere?
> 
> Yes. Although I'd like to get this out so levelbuilders can start developing for an upcoming pilot. I'll add the tests as a follow-up.
>
Sounds good! Is this feature a pilot or mvp?

**Pull request merged**